roles/website/tasks/main.yaml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Add user
+  user: name={{ website_user }} home=/home/{{ website_user }} shell=/bin/zsh
```

Comment:
Ist die Wahl der Shell nicht eher Aufgabe des Server-Administrators? Oder anders gefragt: Wird für irgendwas in dieser Rolle explizit zsh benötigt?